app/grants/templates/grants/activity.html

```diff
@@ -0,0 +1,142 @@
+{% load static humanize i18n grants_extra %}
+<div class="container px-0">
+  <div class="row mb-3 py-2 px-0 mx-sm-0">
+    <div class="col-12 px-0">
+      {% if subscriptions %}
+        <div id="active-subscriptions">
+          <p class="font-body font-weight-semibold subtitle">
+            <i class="g-icon g-icon__dot-circle mr-2"></i>{% trans "Active Subscriptions" %}
+          </p>
+          {% for subscription in subscriptions %}
+            <div class="py-3 mx-sm-0 row transaction-history">
```

Comment:
any way we can DRY up these `transaction-history ` divs?  seems like there is a lot of pasted code in them, and they coudl be one shared template

Reply:
cancelled-subscriptions, active-subscriptions, and  contributions

Reply:
I did wanna do that but when sat with it and realized it becomes a tad bit messy cause 

```
{% for transaction in contributions %}
  {{ transaction.id }} ...
  {{transaction.subscription.X}} ...

```


```
{% for subscription in cancelled_subscriptions %}
  {{subscription.id}} ...
  {{subscription.X}} ...
```

Reply:
have you considered making a subtemplate that has a bunch of variables like so

```
<div>
  <a href="{{url}}">{{title}}</a>
</div>
```

and then calling it like so
```
{% include 'subtemplate.html' with url='foo' title='bar'
```

? 

decoupling the format of the template to variables like above would allow you to DRY and not have a bunch of complex if statements in the template..